plugins/modules/pacemaker_resource.py

```diff
@@ -0,0 +1,238 @@
+        type: list
+        elements: str
+  resource_meta:
+    description:
+      - List of meta to associate with resource.
+    type: list
+    elements: str
+  resource_argument:
+    description:
+      - Actions to associate with resource.
+    type: dict
+    suboptions:
+      argument_action:
+        description:
+          - Action to apply to resource.
+        type: str
+        choices: [ clone, master, group, promotable ]
+      argument_option:
+        description:
+          - Options to associate with resource action.
+        type: list
+        elements: str
+  disabled:
+    description:
+      - Specify argument to disable resource.
+    type: bool
+    default: false
+  wait:
+    description:
+      - Timeout period for polling the resource creation.
+    type: int
+    default: 300
+'''
+
+EXAMPLES = '''
+---
+- name: Create pacemaker resource
+  hosts: localhost
+  gather_facts: false
+  tasks:
+  - name: Create virtual-ip resource
+    community.general.pacemaker_resource:
+      state: create
+      name: virtual-ip
+      resource_type:
+        resource_name: IPaddr2
+      resource_option:
+        - "ip=[192.168.1.2]"
+      resource_argument:
+        argument_action: group
+        argument_option:
+          - master
+      resource_operation:
+        - operation_action: monitor
+          operation_option:
+            - interval=20
+'''
+
+RETURN = '''
+cluster_resources:
+    description: The cluster resource output message.
+    type: str
+    sample: "Assumed agent name ocf:heartbeat:IPaddr2 (deduced from IPaddr2)"
+    returned: always
+'''
+
+from ansible_collections.community.general.plugins.module_utils.module_helper import StateModuleHelper
+from ansible_collections.community.general.plugins.module_utils.pacemaker import pacemaker_runner
+
+
+class PacemakerResource(StateModuleHelper):
+    change_params = ('name', )
+    diff_params = ('name', )
+    output_params = ('status')
+    module = dict(
+        argument_spec=dict(
+            state=dict(type='str', default='present', choices=[
+                'present', 'absent']),
+            name=dict(type='str', required=True),
+            resource_type=dict(type='dict', options=dict(
+                resource_name=dict(type='str'),
+                resource_standard=dict(type='str'),
+                resource_provider=dict(type='str'),
+            )),
+            resource_option=dict(type='list', elements='str', default=list()),
+            resource_operation=dict(type='list', elements='dict', default=list(), options=dict(
+                operation_action=dict(type='str'),
+                operation_option=dict(type='list', elements='str'),
+            )),
+            resource_meta=dict(type='list', elements='str'),
+            resource_argument=dict(type='dict', options=dict(
+                argument_action=dict(type='str', choices=[
+                                   'clone', 'master', 'group', 'promotable']),
```

Comment:
No need to break the line here
```suggestion
                argument_action=dict(type='str', choices=['clone', 'master', 'group', 'promotable']),
```